bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/library/types/DateTimeTypeTest.java

```diff
@@ -143,7 +144,11 @@ public ParameterSet(TimeZone defaultTimeZone, String inputTimeString, String exp
                 { new ParameterSet(TimeZone.getTimeZone("GMT-2"), initTimeMap(), TimeZone.getTimeZone("GMT+3"),
                         "2014-03-30T05:58:47.033-0200") },
                 { new ParameterSet(TimeZone.getTimeZone("GMT-2"), initTimeMap(), TimeZone.getTimeZone("GMT-4"),
-                        "2014-03-30T12:58:47.033-0200") }, });
+                        "2014-03-30T12:58:47.033-0200") },
+                { new ParameterSet(TimeZone.getTimeZone("UTC"), "10:58:47", "1970-01-01T10:58:47.000+0000") },
```

Comment:
Does it work with only "10:58" too? This was the example given in the referenced issue and I reckon it's really important to support it!